ariadne/src/Ariadne/Wallet/Backend.hs

```diff
@@ -26,31 +26,42 @@ createWalletBackend :: WalletConfig -> IO
   )
 createWalletBackend walletConfig = do
   walletSelRef <- newIORef Nothing
+  -- TODO: Do I need to close session on exit?
```

Comment:
Yes, it should use `bracket` style. Maybe we can postpone it until AD-229, though.